src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { Fragment } from 'react';
+import SimpleBarChart from '@carbon/charts-react/bar-chart-simple';
+import StackedBarChart from '@carbon/charts-react/bar-chart-stacked';
+import GroupedBarChart from '@carbon/charts-react/bar-chart-grouped';
+import classnames from 'classnames';
+import every from 'lodash/every';
+import isNil from 'lodash/isNil';
+
+import { BarChartCardPropTypes, CardPropTypes } from '../../constants/PropTypes';
+import {
+  CARD_SIZES,
+  BAR_CHART_TYPES,
+  BAR_CHART_ORIENTATION,
+} from '../../constants/LayoutConstants';
+import Card from '../Card/Card';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const BarChartCard = ({
+  title,
+  content: {
+    xLabel,
+    yLabel,
+    orientation = BAR_CHART_ORIENTATION.VERTICAL,
+    chartType = BAR_CHART_TYPES.SIMPLE,
+    data,
```

Comment:
I'd pass this in as a 'values' prop to be consistent with the other card types and then map it internally to the barchart card data.

Reply:
in addition, we should order the "groups" of data as a new prop of the content object.  I.e. separate the "groups" and the values into two different properties to be consistent with the other card types.

Reply:
refactored to be compatible with the other cards